cortex/utils/commands.py

```
@@ -0,0 +1,344 @@
+
+    # Validate if requested
+    if validate:
+        is_valid, error = validate_command(command, strict=True)
+        if not is_valid:
+            raise CommandValidationError(f"Command validation failed: {error}")
+
+    try:
+        if use_shell:
+            # Shell execution - use with caution
+            # Only allow if command has been validated
+            result = subprocess.run(
```

Ruff (0.14.6)

`subprocess` call with `shell=True` identified, security issue

(S602)